packages/edit-site/src/components/editor/index.js

```diff
@@ -110,14 +110,12 @@ export default function EditSiteEditor( { isLoading } ) {
 			{
 				// Forming a "block formatting context" to prevent margin collapsing.
 				// @see https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Block_formatting_context
-
-				css: `body{${
+				css:
 					canvasMode === 'view'
-						? `min-height: 100vh; ${
+						? `body{min-height: 100vh; ${
 								currentPostIsTrashed ? '' : 'cursor: pointer;'
```

Comment:
I wonder why this UI related CSS is not outside the iframe, while the iframe can itself can be set to `pointer-events:none`.

Reply:
🤔 I'm not sure. It was a [legacy style pulled over in the unification PR](https://github.com/WordPress/gutenberg/pull/62146/files#diff-0475617e1471501c29fbc5c42e2490532cb18793b2c8d1d146fd7bb220ad3542L78)

Looking at the component hierarchy, I'm assuming it could be passed down to `VisualEditor` to `BlockCanvas` via `iframeProps` from the edit-site package.